### PR TITLE
azurerm_redis_linked_server - Adding geo replicated hostname to attribute list

### PR DESCRIPTION
**Pull request by EndureBlackout**
This adds the geo replicated primary hostname to this list of attributes so that it can be used to connect to the primary redis cache and allow users not to have to switch connection strings during failover.

```
=== RUN   TestAccRedisLinkedServer_basic
=== PAUSE TestAccRedisLinkedServer_basic
=== RUN   TestAccRedisLinkedServer_requiresImport
=== PAUSE TestAccRedisLinkedServer_requiresImport
=== CONT  TestAccRedisLinkedServer_basic
=== CONT  TestAccRedisLinkedServer_requiresImport
--- PASS: TestAccRedisLinkedServer_basic (1423.61s)
--- PASS: TestAccRedisLinkedServer_requiresImport (1489.33s)
PASS
```

### CODE REVIEW

File: website/docs/r/redis_linked_server.html.markdown [hunk 1]
```diff
@@ -86,6 +86,8 @@ In addition to the Arguments listed above - the following Attributes are exporte
 
 * `name` - The name of the linked server.
 
+* `geo_replicated_primary_host_name` - The geo hostname for the linked server.
```
Comment (tombuildsstuff) [inline website/docs/r/redis_linked_server.html.markdown:89]: _minor_ could we make this a little more descriptive:

```suggestion
* `geo_replicated_primary_host_name` - The geo-replicated primary hostname for this linked server.
```

File: internal/services/redis/redis_linked_server_resource.go [hunk 2]
```diff
@@ -176,6 +181,7 @@ func resourceRedisLinkedServerRead(d *pluginsdk.ResourceData, meta interface{})
 
 			d.Set("linked_redis_cache_location", location.Normalize(props.LinkedRedisCacheLocation))
 			d.Set("server_role", string(props.ServerRole))
+			d.Set("geo_replicated_primary_host_name", string(*props.GeoReplicatedPrimaryHostName))
```
Comment (tombuildsstuff) [inline internal/services/redis/redis_linked_server_resource.go:184]: Right now this'll crash if a `props.GeoReplicatedPrimaryHostName` isn't returned (i.e. it's `null` in the response) - using `pointer.From` will default this to an empty string when it's unset - so can we update this:

```suggestion
			d.Set("geo_replicated_primary_host_name", pointer.From(props.GeoReplicatedPrimaryHostName))
```

**Review by tombuildsstuff — COMMENTED** [review]
hey @EndureBlackout 

Thanks for this PR - I've left couple of minor comments, but this otherwise LGTM 👍

**Comment by EndureBlackout** [comment]
> hey @EndureBlackout 
> 
> 
> 
> Thanks for this PR - I've left couple of minor comments, but this otherwise LGTM 👍 

Thanks for the feedback! I will get those fixed up when I am back in office.

File: internal/services/redis/redis_linked_server_resource.go (hunk 2 above)
Comment (EndureBlackout) [reply internal/services/redis/redis_linked_server_resource.go]: Done!

**Pull request merged**